apps/web/pages/api/book/event.ts

```diff
@@ -327,20 +327,20 @@ async function handler(req: NextApiRequest) {
 
   if (!users) throw new HttpError({ statusCode: 404, message: "eventTypeUser.notFound" });
 
-  const availableUsers = await ensureAvailableUsers(
-    {
-      ...eventType,
-      users,
-    },
-    {
-      dateFrom: reqBody.start,
-      dateTo: reqBody.end,
-    }
-  );
-
-  // Add an if conditional if there are no seats on the event type
-  // Assign to only one user when ROUND_ROBIN
   if (!eventType.seatsPerTimeSlot) {
```

Comment:
We should only run `ensureAvailableUsers` if there are no seats. Currently since the user that made the event type is counted as "booked" if there is already one attendee.